zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -69,7 +69,7 @@ class EcommerceSpiderParams(BaseSpiderParams):
             "Whether to perform extraction using a browser request "
             "(browserHtml) or an HTTP request (httpResponseBody)."
         ),
-        default=None,
+        default=ExtractFrom.browserHtml,
```

Comment:
Hardcoding this should be okay since we don't have Zyte API's Auto-Configurator yet. So somewhere in the backend, a `None` value is still hardcoded.

Reply:
@BurnzZ what do you think about moving this to BaseSpiderParams? Otherwise I also need to redefine `extract_from` for Articles.

Reply:
Sounds good to me! https://github.com/zytedata/zyte-spider-templates/pull/28/commits/8d7130fe6fd9c7984d76cc0ed040bd7ac006b4b2